op-chain-ops/cmd/receipt-reference-builder/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"os"
+	"time"
+
+	"github.com/mattn/go-isatty"
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+)
+
+const EnvPrefix = "OP_CHAIN_OPS_PROTOCOL_VERSION"
+
+var (
+	FirstFlag = &cli.Uint64Flag{
+		Name:    "first",
+		Value:   0,
+		Usage:   "the first block to include in data collection. INCLUSIVE",
+		EnvVars: opservice.PrefixEnvVar(EnvPrefix, "FIRST"),
+	}
+	LastFlag = &cli.Uint64Flag{
+		Name:    "last",
+		Value:   0,
+		Usage:   "the last block to include in data collection. INCLUSIVE",
+		EnvVars: opservice.PrefixEnvVar(EnvPrefix, "LAST"),
+	}
+	RPCURLFlag = &cli.StringFlag{
+		Name:    "rpc-url",
+		Usage:   "RPC URL to connect to",
+		EnvVars: opservice.PrefixEnvVar(EnvPrefix, "RPC_URL"),
+	}
+	ChainIDFlag = &cli.Uint64Flag{
+		Name:    "chain-id",
+		Usage:   "included in the output file, not consulted for anything else",
+		EnvVars: opservice.PrefixEnvVar(EnvPrefix, "BACKOFF"),
```

Comment:
nit: wrong env var

Reply:
Ah yeah. This flag is actually no longer needed because I just do a call to get chainid